bfxapi/models/position.py

```diff
@@ -19,7 +19,8 @@ class Position:
     """
 
     def __init__(self, symbol, status, amount, b_price, m_funding, m_funding_type,
-                 profit_loss, profit_loss_perc, l_price, lev):
+                 profit_loss, profit_loss_perc, l_price, lev, _, position_id, mts_create, mts_update, type, collateral,
```

Comment:
Looks like you have missed out a placeholder, after `type` and before `collateral`:

```
SYMBOL, 
    STATUS, 
    AMOUNT, 
    BASE_PRICE, 
    MARGIN_FUNDING, 
    MARGIN_FUNDING_TYPE,
    PL,
    PL_PERC,
    PRICE_LIQ,
    LEVERAGE,
    _PLACEHOLDER,
    POSITION_ID,
    MTS_CREATE,
    MTS_UPDATE,
    _PLACEHOLDER,
    TYPE,
    _PLACEHOLDER,
    COLLATERAL,
    COLLATERAL_MIN,
    META,
```